content/en/logs/log_collection/nodejs.md

```diff
@@ -208,4 +208,4 @@ Make sure that the parameter `max_connect_retries` is not set to `1` (the defaul
 [11]: /logs/log_configuration/parsing/?tab=matchers
 [12]: /logs/explorer/#overview
 [13]: https://github.com/winstonjs/winston/blob/master/docs/transports.md#datadog-transport
-[14]: /glossary/#tail
\ No newline at end of file
+[14]: /glossary/#tail
```

Comment:
commited from a github editor, not sure why the github editor plays with the last line